ui/src/styles/_cards.scss

```diff
@@ -56,6 +56,21 @@
     --cardBackground: var(--white);
     display: inline-flex;
   }
+  //digital menus
+  &--pro {
+    color: var(--proCard);
+    li:first-child {
+      color: var(--textColor);
+    }
+    h3 {
+      color: var(--proCard);
+    }
+  }
+  &--expert {
+    h3 {
+      color: var(--expertCard);
+    }
+  }
```

Comment:
couple things here:
1. it feels like there is a disconnect between `color:` and its value of `proCard`. It would be nice to stick with predefined colors (see comment below in vars file)
2. These pro and expert classes could be nice if we had an ongoing concept of pro and expert throughout the site. Maybe in the future it makes sense to do some refactoring and add these in. However, at the moment these are one offs so adding them to the cards file just makes it unnecessarily large. It might be a better idea to use our typography helper classes directly on the html elements for example something like `<h3 className="text-teal">expert ...`